index.js

```diff
@@ -107,18 +109,26 @@ export function checkDataIntegrityProofFormat({
               }
             });
         }
-        it('"proof.created" field MUST exist and be a valid XMLSCHEMA-11 ' +
+        it('if "proof.created" field exists, it MUST be a valid XMLSCHEMA-11 ' +
           'datetime value.', function() {
```

Comment:
This code still says "datetime" instead of "dateTimeStamp" -- please go through the file and change all instances of "datetime" to "dateTimeStamp" (or whatever the spec says).

Reply:
Thanks, those were the only two and I've updated them - [5c31723](https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/16/commits/5c31723b68ea552a895c422d71583dcee759ae48)